test/helpers/story/drivers/dapp-utils/pooltogether.utils.ts

```diff
@@ -0,0 +1,80 @@
+import chai, { expect } from 'chai'
+import { solidity } from 'ethereum-waffle'
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+
+import { IERC20 } from "../../../../../types/typechain"
+import { LoanHelpersReturn } from "../../../loans"
+import LoanStoryTestDriver from '../loan-story-test-driver'
+chai.should()
+chai.use(solidity)
+
+async function lendPoolTogether(
+    hre: HardhatRuntimeEnvironment,
+    loan: LoanHelpersReturn
+  ): Promise<void> {
+    const { contracts } = hre
+    const { details, diamond } = loan
+    const poolTicket = await contracts.get<IERC20>('IERC20', {
+      at: await diamond.getAssetPPoolTicket(details.lendingToken.address),
+    })
+    await diamond
+      .connect(details.borrower.signer)
+      .poolTogetherDepositTicket(
+        details.loan.id,
+        details.loan.lendingToken,
+        details.loan.borrowedAmount
+      )
+
+    const escrowAddress = await diamond.getLoanEscrow(details.loan.id)
+
+    const daiBalance = await details.lendingToken.balanceOf(escrowAddress)
+    daiBalance.eq(details.loan.borrowedAmount).should.eql(false, '')
+
+    const tokenAddresses = await diamond.getEscrowTokens(details.loan.id)
+    tokenAddresses.should.include(poolTicket.address)
+  }
+
+async function withdrawPoolTogether(
+    hre: HardhatRuntimeEnvironment,
+    loan: LoanHelpersReturn
+  ): Promise<void> {
+    const { contracts } = hre
+    const { details, diamond } = loan
+    const poolTicket = await contracts.get<IERC20>('IERC20', {
+      at: await diamond.getAssetPPoolTicket(details.lendingToken.address),
+    })
+    await diamond
+      .connect(details.borrower.signer)
+      .poolTogetherWithdrawAll(details.loan.id, details.lendingToken.address)
+
+    const tokenAddresses = await diamond.getEscrowTokens(details.loan.id)
+    tokenAddresses.should.not.include(poolTicket.address)
+    const escrowAddress = await diamond.getLoanEscrow(details.loan.id)
+    const daiBalance = await details.lendingToken.balanceOf(escrowAddress)
+    daiBalance.should.be.gt('0')
+  }
+
+export const pooltogetherLendTest = async (hre: HardhatRuntimeEnvironment): Promise<void> => {
```

Comment:
man... im sorry this is nit-picky but I cant help but say it.... should be`poolTogetherLendTest` not `pooltogetherLendTest` 😅  right??

Reply:
I have an issue with this, seeing it's same word. But i guess for readability, you're right